PolishWithBlondes/src/App.scss

```diff
@@ -1,12 +1,18 @@
-$primary-color: #3498db;
+* {
+  border: 1px solid red;
+}
 
-.container {
-  background-color: $primary-color;
-  color: white;
-  padding: 20px;
-  text-align: center;
+.app-container {
+  background-color: var(--primary-background);
 }
 
-h1 {
-  font-size: 2rem;
+@media (max-width: 1000px) {
```

Comment:
Not sure if that is possible, but let's try define constant break points and refer to them in media queries

Reply:
Good idea, I'll remember this for the future